charts/opentelemetry-demo/values.yaml

```diff
@@ -737,7 +737,7 @@ grafana:
         - name: Jaeger
           uid: webstore-traces
           type: jaeger
-          url: 'http://{{ include "otel-demo.name" . }}-jaeger:16686'
+          url: 'http://{{ include "otel-demo.name" . }}-jaeger-query:16686/jaeger/ui'
```

Comment:
@puckpuck can you confirm this is needed?

Reply:
@TylerHelmuth I just tested, and your suggestion above is what works: https://otel-demo.honeydemo.io/grafana/datasources/edit/webstore-traces